batch-setup/provision.sh

```diff
@@ -49,3 +49,43 @@ export RAW_TILES_VERSION='%(raw_tiles_version)s'
 export TILEQUEUE_VERSION='%(tilequeue_version)s'
 export VECTOR_DATASOURCE_VERSION='%(vector_datasource_version)s'
 eof
+
+mkdir /tmp/awslogs
+curl 'https://s3.amazonaws.com//aws-cloudwatch/downloads/latest/awslogs-agent-setup.py' -o /tmp/awslogs/awslogs-agent-setup.py
+chmod +x /tmp/awslogs/awslogs-agent-setup.py
+cat > /tmp/awslogs/awslogs.conf <<EOF
+[general]
+state_file = /var/awslogs/state/agent-state
+
+[/var/log/cloud-init-output.log]
+file = /var/log/cloud-init-output.log
+log_group_name = /snapzen/tps
+log_stream_name = {instance_id}
+datetime_format = %%b %%d %%H:%%M:%%S
+EOF
+/tmp/awslogs/awslogs-agent-setup.py -n -r us-east-2 -c /tmp/awslogs/awslogs.conf
```

Comment:
I'm pretty sure that if you echo something in codebuild it'll get saved to a log stream. If not in CloudWatch then in the Codebuild console. Meaning you might not need to set up logs here if you don't want to.

Reply:
You're absolutely right, the stuff that `cron.py` outputs when run by Codebuild is sent to a CloudWatch log stream. However, the `provision.sh` script is run on the TPS instance, which is the long-running EC2 instance managing the import, kicking off batch jobs, etc... As it's a vanilla EC2 instance, it doesn't have logging to CloudWatch on by default, hence the agent setup.